tests/candidateApp-advanced.spec.ts

```diff
@@ -375,35 +404,35 @@ test.describe("when logged in with imported user", () => {
     await page.reload(); //Reload to make sure correct data is loaded to page
 
     // Expect correct title and button
-    await expect(page.getByText(candidateAppTranslationsEn.preview.tip, {exact: true})).toBeVisible();
-    await expect(page.getByRole("button", { name: candidateAppTranslationsEn.preview.close, exact: true })).toBeEnabled();
+    await expect(page.getByText(T.en['candidateApp.preview.tip'], {exact: true})).toBeVisible();
+    await expect(page.getByRole("button", { name: T.en['candidateApp.preview.close'], exact: true })).toBeEnabled();
 
     // Expect correct data for the candidate
     await expect(
       page.getByRole("heading", { name: `${userFirstName} ${userLastName}` }),
     ).toBeVisible();
 
-    await expect(
-      page.getByText(candidateAppTranslationsEn.languages.Finnish, {
-        exact: true,
-      })
-    ).toBeVisible();
-    await expect(
-      page.getByText(`${candidateAppTranslationsEn.languages.Spanish}`, {exact: true})
-    ).toBeVisible();
-
+    for (const lang of [selectedSingleLang, selectedMultiLang[0]]) {
+      await expect(
+        page.getByText(lang, {
+          exact: true,
+        }).first()
```

Comment:
We need `first()` bc there may be multiple elements with the same name of the language as their text.